cmd/gitops-pusher/gitops-pusher.go

```diff
@@ -157,6 +206,10 @@ func main() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+
+	if len(modifiedExternallyFailure) != 0 {
```

Comment:
modified externally? like the file is updated by a human or github?

Reply:
Like the file is updated in the admin panel yeah